wsgi.py

```diff
@@ -361,5 +361,31 @@ def delete_moc_user(user_name):
     )
 
 
+@APP.route("/projects/<project>/quota", methods=["GET"])
+@AUTH.login_required
+def get_quota(project):
+    shift = get_openshift()
+    return Response(
+        response=json.dumps(shift.get_moc_quota(project)),
+        status=400,
```

Comment:
Why is this always returning an error?

Reply:
That should be 200, good catch.  For this initial cut we are always returning what is in the configmap describing all of the quota that can be set.